src/EventTypeDefs.ts

```diff
@@ -192,8 +193,8 @@ type CanvasSelectionEvents = {
 };
 
 export type CollectionEvents = {
-  'object:added': { target: FabricObject };
-  'object:removed': { target: FabricObject };
+  'object:added': { target: StaticFabricObject };
+  'object:removed': { target: StaticFabricObject };
```

Comment:
we need to define this for Canvas/Group with FabricObject

Reply:
yes still a wip not sure what is the best solution here.
I tried to make collection dynamic but i failed, the point is that i should be able to say, something that extends the base object, while is not easy. I tried with interfaces too

Reply:
Ok this makes canvas and static canvas pass the checks, maybe for group we need to refactor again, as soon as we get there we find out